tuftool/README.md

```diff
@@ -159,10 +158,10 @@ for this example we will use a file based url to download from local repo.
 ```sh
 # download tuf repo
 tuftool download \
-   --root "${ROOT}" \
+   --root "${WRK}/tuf-repo/root.json" \
```

Comment:
@SequeI is there any reason why you've changed the root path here ? 
(I know they're identical)